CMake Error at CMakeLists.txt:39 (find_package):   By not providing "FindQt5Gui.cmake" in CMAKE_MODULE_PATH this project has   asked CMake to find a package configuration file provided by "Qt5Gui", but   CMake did not find one.

CMake Error at CMakeLists.txt:39 (find_package):
  By not providing "FindQt5Gui.cmake" in CMAKE_MODULE_PATH this project has
  asked CMake to find a package configuration file provided by "Qt5Gui", but
  CMake did not find one.

  Could not find a package configuration file provided by "Qt5Gui" with any
  of the following names:

    Qt5GuiConfig.cmake
    qt5gui-config.cmake

  Add the installation prefix of "Qt5Gui" to CMAKE_PREFIX_PATH or set
  "Qt5Gui_DIR" to a directory containing one of the above files.  If "Qt5Gui"
  provides a separate development package or SDK, be sure it has been
  installed.

--- a/README.md
+++ b/README.md
@@ -69,7 +69,14 @@ brew install boost
 brew install cmake
 ```
 
-**5. Install Qt 5.9: [download link](http://qt.io/download)**
+**5. Install Qt 5.9:**
+
+  * Ubuntu 16.04 and up
+  ```
+  apt install qtbase5-dev
+  ```
+
+  * Binaries from Qt upstream [download link](http://qt.io/download)
 
 **6. Build**
 

--- a/README.md
+++ b/README.md
@@ -34,7 +34,7 @@ git clone --recurse-submodules https://github.com/ipbc-dev/ipbc-wallet.git
 **2. Install dependencies. For example using Apt:**
 
 ```
-apt-get install -y cmake libboost-dev libboost-date-time-dev libboost-program-options-dev libboost-regex-dev libboost-serialization-dev libboost-system-dev libboost-thread-dev libboost-chrono-dev libboost-filesystem-dev
+apt-get install -y cmake libboost-all-dev qtbase5-dev
 ```
 
 **3. Build**
@@ -69,14 +69,7 @@ brew install boost
 brew install cmake
 ```
 
-**5. Install Qt 5.9:**
-
-  * Ubuntu 16.04 and up
-  ```
-  apt install qtbase5-dev
-  ```
-
-  * Binaries from Qt upstream [download link](http://qt.io/download)
+**5. Install Qt 5.9: [download link](http://qt.io/download)**
 
 **6. Build**
 

